docs/setup/upgrade/devtron-upgrade-0.3.x-0.3.x.md

```diff
@@ -0,0 +1,18 @@
+# Upgrading Devtron 0.3.x to 0.3.x
+
+If you want to check the current version of Devtron you are using, please use the following command.
+
+```
+kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync.data}' | grep "^LTAG=" | cut -d"=" -f2-
+```
+
+## Follow the required step to upgrade the Devtron version
+
+```bash
+helm repo update
+helm upgrade devtron devtron/devtron-operator --namespace devtroncd --set installer.release=v0.3.x
```

Comment:
Split the commands into individual commands or users will just copy entire thing and try to run it as it is which will result in error. 
Using DEVTRON_TARGET_VERSION separately will ensure that users have to give an input here.

-----------------

1. Fetch the latest Devtron helm chart
```bash
helm repo update
```
2. Input the target Devtron version that you want to upgrade to. You can find the latest releases from Devtron on Github https://github.com/devtron-labs/devtron/releases
```bash
DEVTRON_TARGET_VERSION=v0.3.x
```
3. Upgrade Devtron
```bash
helm upgrade devtron devtron/devtron-operator --namespace devtroncd --set installer.release=$DEVTRON_TARGET_VERSION
```